DeepSpeech.ipynb

```diff
@@ -1337,6 +1362,44 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Finally, export the model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "with tf.device('/cpu:0'):\n",
+    "    model_exporter = exporter.Exporter(saver)\n",
+    "    \n",
+    "    #print [n.name for n in session.graph.as_graph_def().node]\n",
+    "    \n",
+    "    inputs = {}\n",
+    "    outputs = {}\n",
+    "    for i in xrange(len(available_devices)):\n",
+    "        # Create output for tower i\n",
+    "        input = session.graph.get_tensor_by_name('tower_%d/input_batch:0' % i)\n",
+    "        output = session.graph.get_tensor_by_name('tower_%d/output_decoded:0' % i)\n",
```

Comment:
So I guess the operator will be 'scope/name' and the resulting tensor will be 'scope/name:0'. These tensors exist, you get assertions if they don't, or if they're operators and not tensors, and I verified them while I was writing it.